src/Components/People.js

```diff
@@ -0,0 +1,104 @@
+import axios from "axios";
+import { useState } from "react";
+
+const People = () => {
+	// declare the state
+	const [userInput, setUserInput] = useState("");
+	const [person, setPerson] = useState([]);
+	const [personAge, setPersonAge] = useState("");
+	const [personGender, setPersonGender] = useState("");
+	const [personList, setPersonList] = useState({});
+
+	// take in the event
+	const handleChange = (e) => setUserInput(e.target.value);
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const res = await axios.get("https://ghibliapi.herokuapp.com/people");
+			setPersonList(res.data);
+
+			setPerson(res.data.filter((person) => person.name === userInput)[0].name);
+			setPersonAge(
+				res.data.filter((person) => person.name === userInput)[0].age
+			);
+			setPersonGender(
+				res.data.filter((person) => person.name === userInput)[0].gender
+			);
+		} catch (err) {
+			// setPersonList({});
+		}
+		setUserInput("");
+	};
+
+	// if (!personList.name.includes(userInput)) {
+	// 	<p>Not Found</p>
```

Comment:
You need to handle this case in order to pass the last test. I suggest having a ternary in the HTML portion of the code